lib/rules/max-statements-per-line.js

```diff
@@ -35,11 +35,12 @@ module.exports = {
         const sourceCode = context.getSourceCode(),
             options = context.options[0] || {},
             maxStatementsPerLine = typeof options.max !== "undefined" ? options.max : 1,
-            message = "This line has too many statements. Maximum allowed is {{maxStatementsPerLine}}.",
-            data = { maxStatementsPerLine };
+            emptyBlockMessage = "This line has an empty block, but no statements are allowed.",
```

Comment:
What I'm saying is that we will remove that special case completely so no one will see that. Nothing special needs to be done because it's going away.